roles/deploy/hooks/finalize-after.yml

```diff
@@ -13,6 +13,18 @@
     chdir: "{{ deploy_helper.new_release_path }}"
   when: wp_installed.rc == 0
 
+- name: Update WP database
+  command: wp core update-db
+  args:
+    chdir: "{{ deploy_helper.new_release_path }}"
+  when: wp_installed.rc == 0 and not (project.multisite.enabled | default(False) == True)
+
+- name: Update WP network database
+  command: wp core update-db
```

Comment:
Musta gotten lost in a copy/paste.

Added a message. Let me know if you have a preference on the copy.